src/lib.rs

```diff
@@ -42,7 +42,7 @@
 //!
 //! ## Rust Version
 //!
-//! This version of itertools requires Rust 1.32 or later.
+//! This version of itertools requires Rust 1.34 or later.
```

Comment:
If your assessment is correct, we could possibly increment the minimum rust version in a separate commit?

Reply:
Do you mean a pull request? It's already a separate commit.